source/extensions/stat_sinks/well_known_names.h

```diff
@@ -18,6 +18,8 @@ class StatsSinkNameValues {
   const std::string DOG_STATSD = "envoy.dog_statsd";
   // MetricsService sink
   const std::string METRICS_SERVICE = "envoy.metrics_service";
+  // Hystrix sink
+  const std::string HYSTRIX = "envoy.hystrix";
```

Comment:
See Note above, please call this "envoy.stat_sinks.hystrix" (and update any necessary docs)